File: hw/medina/u2/hw19Inheritance/pythonFarmSystem/Cage.py
class Cage:
    def __init__(self, id, description, type_cage):
        self._id = id
        self._description = description
        self._type = type_cage  # 1 coop, 2 stable, 3 pens
        self._location = None

    def __str__(self):
        return f"Cage\nid.: {self._id}\ndescription: {self._description}\ntype: {self._type}"

    def get_id(self):
        return self._id

    def set_id(self, id):
        self._id = id

    def get_description(self):
        return self._description

    def set_description(self, description):
        self._description = description

    def get_type(self):
        return self._type

    def set_type(self, type_cage):
        self._type = type_cage

File: hw/rojas/u2/hw19Inheritance/Farmsystem/src/main/java/ec/edu/espe/farmsystem/model/Cage.java
package ec.edu.espe.farmsystem.model;
/**
 *
 * @author Josue Rojas
 */

public class Cage {
    private int id;
    private String description;
    private int type; //1 coop, 2 stable, 3 pens
    Location location;

    @Override
    public String toString() {
        return "Cage" + "\nid.: " + id + "\ndescription: " + description + "\ntype: " + type ;
    }

    public Cage(int id, String description, int type) {
        this.id = id;
        this.description = description;
        this.type = type;
    }
    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * @return the type
     */
    public int getType() {
        return type;
    }

    /**
     * @param type the type to set
     */
    public void setType(int type) {
        this.type = type;
    }
    
    
    
    
    
    
    
}
